src/Libraries/RevitNodes/Elements/CurtainPanel.cs

```diff
@@ -238,7 +248,9 @@ public bool IsRectangular
             return result;
          }
       }
-
+       /// <summary>
+       /// Gets the width of the specific curtain panel
```

Comment:
In this method, there are few hardcoded strings used, can you please move them to the resource file.